entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+set_config() {
+    KEY=$1
+    VALUE=$(eval echo \$${KEY})
+    if [ -z "${VALUE}" ]; then
+        echo "${KEY} must be set" 1>&2
+        exit 1
+    fi
+    sed -i "s/{{${KEY}}}/${VALUE}/g" /nginx.conf
+}
+
+set_config HOSTNAME
+for i in BLDSTATUS CCUTTER LTDSTATUS METRICDEV STATUS; do
+    for j in HOST PORT; do
```

Comment:
⚡️  For writing a nested for loop in bash that doesn't make me want to link the Vader Ocean Brita meme.